src/Chapter2.hs

```diff
@@ -783,10 +813,10 @@ mastered the skill of eta-reducing.
 divideTenBy :: Int -> Int
 divideTenBy x = div 10 x
```

Comment:
I noticed, that you didn't complete eta-reducing task, so here is a small note about that. Hope it would help with the understanding:

When eta-reducing, you need to remove arguments from both sides of the `=` sign. So the correct solution is:

```haskell
divideTenBy = div 10
```

See how it compares with the original code:

```diff
-divideTenBy x = div 10 x
+divideTenBy = div 10
```

The motivation behind eta-reduction if that if you have two functions `foo` and `bar` of the same type, and they return the same results for any arguments (so `foo x` always equal to `bar x` for any `x`). In such situations, we say that functions `foo` and `bar` are equal, and if we want to define `foo` in terms of `bar`, we can write `foo = bar` instead of a longer `foo x = bar x` (which means that `foo` returns the same result as `bar` for any `x`).

Does it make sense?

Reply:
I get it now. Thanks!